test/path-test.js

```diff
@@ -1,349 +1,296 @@
-var tape = require("tape"),
-    path = require("../");
+import assert from "assert";
+import _pathEqual from "./pathEqual.js";
+const pathEqual = _pathEqual(assert);
+import {path} from "../src/index.js";
 
-require("./pathEqual");
-
-tape("path is an instanceof path", function(test) {
-  var p = path.path();
-  test.ok(p instanceof path.path);
-  test.pathEqual(p, "");
-  test.end();
+it("path is an instanceof path", () => {
+  var p = path();
```

Comment:
If you’re willing, it might be a good time to at least rewrite the tests for const/let, since there’s already so much churn.

```suggestion
  const p = path();
```